helm/grafana/Chart.yaml

```diff
@@ -1,28 +1,27 @@
 apiVersion: v2
-appVersion: 7.3.5
+name: grafana
+version: 6.24.1
```

Comment:
Why are we moving from `0.3.0` to `6.24.1` it seems a fairly large jump, what's the reasoning here ?

Reply:
No reasonning apart that 0.3.0 is our own numbering :(

Reply:
I need to put that to 0.4.0